Source/MQTTnet/Server/MqttServer.cs

```diff
@@ -207,7 +207,15 @@ public Task<IList<MqttSessionStatus>> GetSessionsAsync()
             return _clientSessionsManager.GetSessionStatusAsync();
         }
 
-        public Task InjectApplicationMessage(InjectedMqttApplicationMessage injectedApplicationMessage, CancellationToken cancellationToken = default)
+        public Task InjectApplicationMessage(
+            InjectedMqttApplicationMessage injectedApplicationMessage,
+            CancellationToken cancellationToken = default) =>
+            InjectApplicationMessage(
+                injectedApplicationMessage,
+                _sessionItems,
+                cancellationToken);
+
+        public Task InjectApplicationMessage(InjectedMqttApplicationMessage injectedApplicationMessage, ConcurrentDictionary<object, object> sessionItems, CancellationToken cancellationToken = default)
```

Comment:
I don't like the idea to pass in session items for every injected message. The server should behave like a single client. 

We may need to expose the session items as a public property for _MqttServer_. It is currently missing the feature to setup session items externally. This is usually done for clients when they connect but the server has no connection at all so we must prepare the session items somewhere else.

Reply:
As a part of #1635 I added the ability to get session items using the IMqttServerExtensibility interface. Maybe this is a way of exposing it without opening up a SessionItems dictionary to all? This also allows users to supply whatever session items they want by calling directly to the MqttClientSessionsManager.